content/blog/2019/09/2019-09-16-outreachy-audit-log-release.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Audit Log Plugin for Jenkins Releases 1.0"
+tags:
+- outreachy
+- logging
+- security
+- plugin
+- community
+author: jvz
+---
+
+Thanks to our Outreachy interns over the past year, I'm proud to announce the initial release of the https://plugins.jenkins.io/audit-log[Audit Log plugin for Jenkins].
+This plugin is the first major project completed related to Outreachy, and I'd like to give a brief overview of the functionality that was developed for this release.
+The primary goal of this plugin is to introduce an audit trail of various Jenkins events using structured logging and related audit logging standards.
+Initially, this plugin covers audit events related to core Jenkins concepts like user accounts, jobs, builds, nodes, and credentials usage.
+More specifically, this tracks user login and logout events, credentials usage, user creation (when using the Jenkins user database as a security realm), user password updates (ditto), starts and ends of builds, creation/modification/deletion/copying of items (which correspond to projects, pipelines, folders, etc.), and the creation/modification/deletion of nodes.
```

Comment:
Might be nice to list what is being tracked in bullet point list (this is just an optimization for legibility/scan reading :)